containers/podman-desktop
# chore: add copy to clipboard button to resources page

### What does this PR do?

This PR adds a new button to the resources page. The added button provides the user with a way to copy the provider endpoint to the clipboard.

### Screenshot / video of UI

<img width="1184" alt="Screenshot 2024-01-25 at 11 17 52" src="https://github.com/containers/podman-desktop/assets/16507629/63f35123-3a03-4eaf-84a5-16fc3e935088">

### What issues does this PR fix or reference?

Fixes #5608 

### How to test this PR?

1) Run Podman Desktop from this branch
2) Go to Settings > Resources
3) Check if the copy icon stands next to the resource endpoint
4) Click on it
5) Verify if the endpoint text is on your clipboard
